apps/cedille/base/deployment.yaml

```diff
@@ -1,21 +1,43 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cedille-service
+  name: cedille
+  annotations:
+    kube-score/ignore: container-image-pull-policy,container-image-tag, pod-networkpolicy
 spec:
   replicas: 1
```

Comment:
lets add two replicas. Ive noticed when we update the website there's some downtime